src/odemis/acq/stitching/_tiledacq.py

```diff
@@ -1081,11 +1112,12 @@ def estimate_time(self, roi_idx=0, actual_time_per_roi=None) -> float:
             acquisition_time = actual_time_per_roi * remaining_rois
```

Comment:
There are now 2 different ways to report the time left. The old "actual_time_per_roi" way, which assumes every RoI is going to take on average the same time, and the new "time_per_task" way, which updates the progress according to the subfutures progress. 
I don't think you need to keep the old way. Especially, it's possible it's going to work against the new way, causing weird fluctuation in the expected time. (Of course, right now this function is probably  never called with more than 1 RoI, so it's not noticeable).

=> remove everything related to the "actual_time_per_roi". That's going to simplify the code quite a bit, and avoid odd fluctuations.

Reply:
I understand that the two methods are working against each other. Should I be keeping it same as before and remove the new changes or keep the new changes?

Reply:
Remove the old way.